.gitignore

```diff
@@ -5,3 +5,6 @@ node_modules
 npm-debug.log
 # WebStorm IDE files
 .idea/*
+*.iml
+
+*swp
```

Comment:
:q  to exit.  I know how you feel.  I once accidentally opened "ed" and had to shut off the computer :P